backend/.env.example

```
@@ -12,3 +12,9 @@ DB_CONNECTION_STRING=[db_client]://[db_username]:[db_user_password]@localhost:[d
 DB_DIALECT=pg
 DB_POOL_MIN=2
 DB_POOL_MAX=10
+
+#
+# SENDGRID
+#
+EMAIL_API_KEY=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
```

```suggestion
EMAIL_API_KEY=key
```

we will send the key to the rest of the team in chat